output/crackls80/log.txt

```diff
@@ -0,0 +1,32 @@
+[22:01:31.646] Namespace(root_path='datasets/CrackLS315', num_classes=1, output_dirs='output/crackls315', max_epochs=100, batch_size=16, n_gpu=2, deterministic=1, base_lr=0.001, img_size=[512, 512], seed=44, cfg='configs/config.yaml', resume=None, accumulation_steps=None, cache_mode='part', opts=None, zip=False, use_checkpoint=False, amp_opt_level='O1', tag=None, eval=False, throughput=False)
```

Comment:
why do we need this?

Reply:
It's like a checkpoint to see which epoch exactly did the code stop running, or from where the model started getting overfit on the data, and so on.

Reply:
OK, I assume there will be more like this in the future? Then you may want to have a log name with some date info,

Reply:
Yes, that's right. Since the training is not able to run on my system, due to insufficient GPU memory there is only one for now.